src/pages/home/report/ReportActionsView.js

```diff
@@ -125,6 +140,9 @@ class ReportActionsView extends React.Component {
         // The last sequenceNumber of the same report has changed.
         const previousLastSequenceNumber = lodashGet(lastItem(prevProps.reportActions), 'sequenceNumber');
         const currentLastSequenceNumber = lodashGet(lastItem(this.props.reportActions), 'sequenceNumber');
+        const shouldRecordMaxAction = Visibility.isVisible()
+            && !(this.props.isDrawerOpen && this.props.isSmallScreenWidth);
```

Comment:
This logic is a bit hard to follow... can we please get rid of the `!(this && that)` logic so that this more accurately presents which scenarios we want to mark the report as read?

Reply:
I am not following your comment here. This check make sure that we don't mark the actions as read when drawer is open on small screens.

On the other hand, Drawer is always open on web so we want to let it read.

Reply:
Might be a personal preference, but it took me some time to parse

e.g. we were to explain this to someone we might say we want to mark as read in two situations 

1. The app is visible and we are not on a small screen

OR

2. The app is visible, we are on a small screen, and the drawer is closed

Compare this to saying we want to mark as read when:

1. The app is visible AND all other situations except when the drawer is open and the screen is small.

Maybe just me, but this is easier to understand if you see it like this...

```js
// Always record on large screens. Only record on small screens when the sidebar is closed.
const shouldRecordMaxAction = Visibility.isVisible() 
    && (!this.props.isSmallScreenWidth || !this.props.isDrawerOpen)
```

lmk if it looks the same to you 😂

Reply:
Yup. Will update.